charts/incubator/clickhouse/values.yaml

```diff
@@ -0,0 +1,70 @@
+image:
+  repository: yandex/clickhouse-server
+  pullPolicy: IfNotPresent
+  tag: 21.3.20.1@sha256:4eccfffb01d735ab7c1af9a97fbff0c532112a6871b2bb5fe5c478d86d247b7e
+
+controller:
+  type: statefulset
+  strategy: RollingUpdate
+  rollingUpdate:
+    unavailable: 1
+
+securityContext:
+  capabilities:
+    add:
+      - IPC_LOCK
+      - NET_ADMIN
+      - SYS_NICE
```

Comment:
Sorry but a database should never need this.
Why the heck is this app requesting this level of access for a database?

Reply:
No idea why, just adhering to the docs.

https://github.com/ClickHouse/ClickHouse/blob/master/docker/server/README.md#linux-capabilities

Reply:
Not really relevant now though, as it's not starting anyway...